### PR TITLE
Format gfx platform #21373

### PR DESCRIPTION
**Pull request by kingdido999**
```bash
rustfmt $(find components/gfx/platform -type f -name "*.rs")
```

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21610)

### CODE REVIEW

**Comment by kingdido999** [comment]
Fixed.

**Comment by tigercosmos** [comment]
please squash commits

File: components/gfx/platform/freetype/font_template.rs [hunk 1]
```diff
@@ -15,17 +15,18 @@ use webrender_api::NativeFontHandle;
...
 impl fmt::Debug for FontTemplateData {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         fmt.debug_struct("FontTemplateData")
-           .field("bytes", &self.bytes.as_ref().map(|b| format!("[{} bytes]", b.len())))
-           .field("identifier", &self.identifier)
-           .finish()
+            .field(
+                "bytes",
+                &self.bytes.as_ref().map(|b| format!("[{} bytes]", b.len())),
+            ).field("identifier", &self.identifier)
```
Comment (nox) [inline components/gfx/platform/freetype/font_template.rs]: Shouldn't the next `.field` call start on a new line? oO

Comment (kingdido999) [reply components/gfx/platform/freetype/font_template.rs]: I'll file a bug to rustfmt 😢

Comment (kingdido999) [reply components/gfx/platform/freetype/font_template.rs]: Looks like a similar issue has been reported: https://github.com/rust-lang-nursery/rustfmt/issues/2932

**Pull request merged**